avocado/core/runner.py

```diff
@@ -339,6 +339,10 @@ def run_test(self, test_factory, queue, summary, job_deadline=0):
         """
         proc = None
         sigtstp = multiprocessing.Lock()
+        if 'modulePath' in test_factory[1]:
+            test_path = test_factory[1]['modulePath']
+            test_module_dir = os.path.abspath(os.path.dirname(test_path))
+            sys.path.insert(0, test_module_dir)
```

Comment:
Naming the test modules after system libraries would be a bad idea in the first place.

Now there is merit to wanting to isolate the test process from the main avocado process, I agree with the concern from that perspective. I will take a look at this later then.